dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -418,35 +443,46 @@ def test_check_meta():
         check_meta(df2, meta2, funcname="from_delayed")
     frame = "pandas.core.frame.DataFrame" if not PANDAS_GE_300 else "pandas.DataFrame"
 
-    exp = (
-        "Metadata mismatch found in `from_delayed`.\n"
-        "\n"
-        f"Partition type: `{frame}`\n"
-        "+--------+----------+----------+\n"
-        "| Column | Found    | Expected |\n"
-        "+--------+----------+----------+\n"
-        "| 'a'    | object   | category |\n"
-        "| 'c'    | -        | float64  |\n"
-        "| 'e'    | category | -        |\n"
-        "+--------+----------+----------+"
+    if PANDAS_GE_300:
```

Comment:
This mixes reformatting with actual changes, sorry. I reformatted it to use `textwrap.dedent`. The only meaningful change is the value used for the string dtype (str or object).